src/components/ShowUser/ShowUser.jsx

```diff
@@ -10,11 +11,10 @@ class ShowUser extends Component {
 	}
 
 	componentDidMount() {
-		fetch(`http://${process.env.REACT_APP_API_URI}:${process.env.REACT_APP_API_PORT}/api/users/${this.props.match.params.id}`)
-			.then(res => res.json())
+		fetch(userService.getById(this.props.match.params.id)
 			.then((data) => {
 				this.setState({ user: data });
-			})
+			}))
 			.catch(console.log);
```

Comment:
brug det alert gøjl som andreas har brugt i catchen istedet for at console.log
mener den hedder toastify